config/initializers/assets.rb

```diff
@@ -11,4 +11,4 @@
 # Precompile additional assets.
 # application.js, application.scss, and all non-JS/CSS in the app/assets
 # folder are already added.
-# Rails.application.config.assets.precompile += %w( admin.js admin.css )
+# Rails.application.config.assets.precompile += %w( admin_features.js admin_features.css )
```

Comment:
Why is this relevant?

Reply:
Dont know, this is just autogenerated with active admin. Since its JS and CSS files I suspect its something to do with the layout of the dashboard.

Reply:
But did you do any changes to it yourself?

Reply:
No I haven't done any changes to it myself.